src/edu/macalester/comp127/FarmArt/BackgroundBrush.java

```diff
@@ -0,0 +1,21 @@
+package edu.macalester.comp127.FarmArt;
+
+/**
+ * a background brush that implements brush interface
+ */
+public class BackgroundBrush implements Brush{
+    private final BackgroundType background;
+
+    /**
+     * constructor for element brush
+     * @param background
+     */
+    public BackgroundBrush(BackgroundType background) {
+        this.background = background;
+    }
+
+    @Override
+    public void apply(Tile tile) {
+        background.apply(tile);
```

Comment:
Here and in the `ElementBrush`, it feels a little odd that the background applies itself to the tile. It leaves the two classes a bit too much mutually entangled for my taste. I’d consider instead:
```java
tile.setBackground(background.generateColor());
```